packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -848,11 +888,13 @@ export default decorate([
                   {_('newBackupSettings')}
                   <ActionButton
                     className='pull-right'
-                    data-mode='displayAdvancedSettings'
+                    data-mode='_displayAdvancedSettings'
```

Comment:
What's the purpose of this attribute?

Reply:
Ok, I understand.

Reply:
I don't think this works correctly, what if `_displayAdvancedSettings` is `false` but `displayAdvancedSettings` is `true`

Reply:
It means that the user don't want display the settings.

Reply:
My bad it will not works